lib/kubernetes-deploy/kubernetes_resource/resource_quota.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class ResourceQuota < KubernetesResource
+    TIMEOUT = 30.seconds
+
+    def sync
+      _, _err, st = kubectl.run("get", type, @name)
+      @status = st.success? ? "Available" : "Unknown"
+      @found = st.success?
+    end
+
+    def deploy_succeeded?
+      exists?
```

Comment:
I just noticed that this resource type [has a status](https://kubernetes.io/docs/api-reference/v1.6/#resourcequota-v1-core): `Status defines the actual enforced quota and its current usage`. Based on your experience, do you think it would be reasonable to define success as `spec.hard == status.hard`?

Reply:
I think that would work. I am not entirely sure if its guaranteed to be in the same units though